packages/ibm-products/src/components/Guidebanner/Guidebanner.docs-page.js

```diff
@@ -12,6 +12,12 @@ import * as stories from './Guidebanner.stories';
 
 export const DocsPage = () => (
   <StoryDocsPage
+    altGuidelinesHref={[
+      {
+        href: 'https://pages.github.ibm.com/carbon/ibm-products/components/onboarding/guide-banner/usage/',
+        label: 'Guide banner usage guidelines',
```

Comment:
Since the published docs did end up using “guide banner” ... I’m inclined to believe this should be named `GuideBanner` after all. 😜 

For a future discussion, I suppose. 😅

Reply:
There was a lot of back-and-forth about this during design and development.

My suggestion would be that the component is "GuideBanner" and the documentation is "Guide banner". We have the same discrepancies with the other components: "AboutModal / About modal", "OptionsTile / Options tile", etc.

Reply:
Haha, I just remember we discussed whether it should be guidebanner (`Guidebanner`) (like tearsheet) or guide banner (like side panel) last year. But looks like we should have stood our ground on it being guide banner (`GuideBanner`). 🫠